src/compiler/sexp.rs

```diff
@@ -640,7 +640,7 @@ fn parse_sexp_step(loc: Srcloc, p: &SExpParseState, this_char: u8) -> SExpParseR
                             emit(Rc::new(new_tail), SExpParseState::Empty)
                         } else {
                             list_copy.push(Rc::new(new_tail));
-                            let new_list = enlist(pl.ext(l), list_copy);
```

Comment:
Where is `l` declared? I'm not seeing it yet.

Reply:
SExpParseState::Bareword(l, t)

Reply:
line 633